app/views/medications/_form.html.erb

```diff
@@ -1,10 +1,5 @@
 <div id="medication_content">
   <%= form_for(@medication) do |f| %>
-  <% if @medication.errors.any? %>
```

Comment:
Also to avoid duplication, should we just include `<%= render 'validation_errors' %>` here instead of in `app/views/medications/edit.html.erb` and `app/views/medications/new.html.erb`?

Reply:
Hi Julia! I thanks for the code review and comments!

I tried this and unfortunately it does not work in this case. `@medication` in line ones does not have any errors because of the else block of the create action in the medications controller.